JSDemos/Demos/Toolbar/Multiline/description.md

```diff
@@ -1,5 +1,5 @@
-## Configure Toolbar Items
+This demo demonstrates the multiline mode in the Toolbar component. To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 
```

Comment:
I don't know if we need the second sentence. I can always see the tree in these demos, I know all components have multiple associated demo modules. So there's probably always going to be a basic demo. Should we include such references in each demo? 

If you do want to keep it, then maybe I would have it as a separate paragraph.

Reply:
I think it's better to link the tutorial in the demo, since the "basic" demo doesn't have all the info to get started with a component.